docs/source/getting_started/getting_started_index.rst

```diff
@@ -0,0 +1,13 @@
+Getting Started
+---------------
+
+How to start working with Featuretools; The main concepts
```

Comment:
Would it make sense to add a link back to the 5 minute quick start here? Something like:
> For a quick introduction to Featuretools, check out our [5 minute quick start guide](https://docs.featuretools.com/en/pydata-sphinx-theme/index.html#minute-quick-start).
>
> How to start working with Featuretools; the main concepts:

Reply:
Good idea